src/components/App/App.js

```diff
@@ -1,75 +1,79 @@
-import Dashboard from '../Dashboard/Dashboard'
-import UsersContext from '../../Context/UsersContext'
-import ErrorContext from '../../Context/ErrorContext'//Probably don't need an error context because App can conditionally render error if it's state exists
-import RequestRoomieContext from '../../Context/RequestRoomieContext'
-import CurrentUserContext from '../../Context/CurrentUserContext'
-import { getUsers } from '../../utilities/apiCalls'
+import React, { useState, useEffect } from "react"
+import { Switch, Route } from "react-router-dom"
+import Navbar from "../Navbar/Navbar"
+import Home from "../Home/Home"
 
-export default function App() {
+import dummyData from "../../utilities/dummyData"
 
-  const [users, setUsers] = useState([])
-  const [error, setError] = useState('')
-  const [currentUser, setCurrentUser] = useState(
-    {
-      "id": "13",
-      "type": "user",
-      "attributes": {
-        "roomies": [{id: '5', status: 'pending'}],
-        "email": "dusty@email.com",
-        "name": "The Dust",
-        "gender": "male",
-        "age": 90,
-        "location": {
-        "city": "Orlando",
-        "state": "FL"
-        }
-      }
-    }
-  )
+import "./App.scss"
+import Dashboard from "../Dashboard/Dashboard"
+import Login from "../Login/Login"
+import UsersContext from "../../Context/UsersContext"
+import ErrorContext from "../../Context/ErrorContext"
+import RequestRoomieContext from "../../Context/RequestRoomieContext"
+import CurrentUserContext from "../../Context/CurrentUserContext"
+import { getUsers } from "../../utilities/apiCalls"
 
+export default function App() {
+  const [users, setUsers] = useState([])
+  const [error, setError] = useState("")
+  const [currentUser, setCurrentUser] = useState(null)
 
   useEffect(() => {
     const fetchUsers = async () => {
       try {
-        const response = await getUsers();
-        setUsers(response.data);
+        const response = await getUsers()
+        setUsers(response.data)
       } catch (err) {
-        setError(err);
+        setError(err)
       }
-    };
-    fetchUsers();
-  }, []);
+    }
+    // fetchUsers()
+    // Commented out to use dummy data
 
+    setUsers(dummyData.data)
+  }, [])
 
-  const requestRoomie = async id => {
+  const requestRoomie = async (id) => {
     // const updatedCurrentUser = await(handleRoomieRequest)
     // setCurrentUser(response)
-    setCurrentUser(prevUser => ({
+    setCurrentUser((prevUser) => ({
       ...prevUser,
-      attributes: {...prevUser.attributes, roomies: [ ...prevUser.attributes.roomies, {id, status: 'pending'} ]}
+      attributes: {
+        ...prevUser.attributes,
+        roomies: [...prevUser.attributes.roomies, { id, status: "pending" }],
+      },
     }))
   }
 
   return (
     <CurrentUserContext.Provider value={currentUser}>
       <RequestRoomieContext.Provider value={requestRoomie}>
-        <ErrorContext.Provider value={error}>
+        {/* <ErrorContext.Provider value={error}> */}
```

Comment:
We will have to see if we will use the error property in state anywhere else in the code or not, otherwise can just delete this context altogether.